src/test/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/ActionGroupsApiTest.java

```diff
@@ -298,4 +307,36 @@ public void testActionGroupsApi() throws Exception {
         Assert.assertEquals(1, permissions.size());
         Assert.assertTrue(permissions.contains("READ_UT"));        
 	}
+
+    @Test
+    public void testActionGroupsApiWithoutCertificate() throws Exception {
```

Comment:
Should we rename it to testActionGroupsApiForNonSuperAdmin() ?

Reply:
Make sense. Changed.